uktrade/data-hub-frontend
# Canadian Postcode Valiation

## Description of change

This PR adds format validation to the Canadian postcode field on both the add company and edit company forms.

## Test instructions

Attempt to add or edit a Canadian company and change the postcode to something that does not match this format: https://en.wikipedia.org/wiki/Postal_codes_in_Canada
You should receive an error that the Postal Code format is invalid. If you correct this, the error should be gone.

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/20663545/141802981-98ceb7c5-95ad-4130-8ef8-48b290b46d49.png)
There is no error on entry of an invalid Postal Code

### After

![image](https://user-images.githubusercontent.com/20663545/141803104-b5dd4fbd-dd43-4e12-975a-8a71b661663b.png)
There is now an error on entry of an invalid Postal Code

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
